src/test/lib/operators/limit_test.cpp

```diff
@@ -147,4 +147,14 @@ TEST_F(OperatorsLimitTest, ForwardSortedByFlag) {
   }
 }
 
+TEST_F(OperatorsLimitTest, OnlyIntegralTypes) {
+  auto limit = std::make_shared<Limit>(_table_wrapper, to_expression(AllTypeVariant{"test_string"}));
```

Comment:
const? `value_("test_string")` rather than `to_expression(...)`?

Reply:
also: The name says only integral, but I only see a string value in the test. Add another case with a floating-point?